docs/how-to-proofread-files.md

```diff
@@ -0,0 +1,43 @@
+# How to Proofread Translations
+
+Our proofreading team is responsible for ensuring that translations are accurate and do not add, modify, or remove content from the text.
```

Comment:
We went over this pretty heavily in the translation meetings - it might be worth adding a section about translating the content while preserving our intent and difficulty, though. @RandellDawson thoughts?